pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Description 📝
+
+Please provide a brief description of the purpose and goals of this pull request.
+
+## Changes 💬
+
+- High-level overview of the changes introduced.
+- Step-by-step breakdown of the implementation, if applicable.
+
+## Checklist ✅
+
+- [ ] Reviewed the code changes thoroughly.
+- [ ] Tested the changes locally and ensured they work as expected.
+- [ ] Updated relevant documentation.
+- [ ] Added appropriate test cases, if applicable.
+- [ ] Considered potential security or performance implications.
+- [ ] Code adheres to the project's style guidelines.
+- [ ] Addressed all reviewer comments and concerns, if any.
+
+## Screenshots (if applicable)
+
+Include relevant screenshots or GIFs demonstrating the changes, especially for UI-related updates.
+
+## Additional Notes 📒
+
+Include any additional information or context that might be helpful for the reviewers. 
+***Also if you are fixing the already made PR and the reviewer demands some change in it then, in each PR you have to mention the issue you have fixed in it.***
```

Comment:
Not like this. It should be general and it should come in the `Description` section.

We can use something like this:
```
Fixes/Closes: <issue-link>
```

Reply:
got it, so you are saying me to do something like this?? 👇
![Screenshot 2023-05-29 at 18-08-35 Editing GyanaGuru_pull_request_template md at pull_request_template · DSR1729_GyanaGuru](https://github.com/PiyushKalyanpy/GyanaGuru/assets/115932772/80cf1d56-a003-4272-99dd-8f832b8b35db)

Reply:
move the `Fixes/Close: <issue/link>` one line up

Reply:
Is this fine ?
![Screenshot 2023-05-29 at 21-48-32 Editing GyanaGuru_pull_request_template md at pull_request_template · DSR1729_GyanaGuru · GitHub](https://github.com/PiyushKalyanpy/GyanaGuru/assets/115932772/c0fc8b8a-b9df-4b1c-a74b-b185319ea3ee)

Reply:
I meant `Fixes/Closes: <issue-link>` in one line after the description. Like this:
```markdown
## Description 📝

<!-- NOTE: Remove the one which doesn't apply to your PR. If your PR is fixing something then remove `/Closes` -->
Fixes/Closes: <issue-link>
Please provide a brief description of the purpose and goals of this pull request.
```